### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by MatsalakO**

### CODE REVIEW

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,36 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_check_human_age_when_animal_age_less_than_15() -> None:
+    result = get_human_age(14, 14)
```
Comment (pavlejviki) [inline app/test_main.py:5]: no need to create additional variable. remove in the rest of the tests as well

**Review by pavlejviki — CHANGES_REQUESTED** [review]

File: app/test_main.py [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+def test_check_human_age_when_animal_age_equal_27() -> None:
+    assert get_human_age(27, 27) == [2, 2]
+
+
+def test_check_human_age_when_cat_and_dog_age_equal_28() -> None:
+    assert get_human_age(28, 28) == [3, 2]
+
+
+def test_check_human_age_when_cat_and_dog_age_equal_100() -> None:
+    assert get_human_age(100, 100) == [21, 17]
```
Comment (Dimosphen1) [inline app/test_main.py:29]: Rewrite your tests using `pytest` as mentioned in the task requirements. Besides that, include tests to check cases when incoming cat and dog years differ one from another or include negative numbers. Also test for raising exception when incorrect data types are provided as mentioned [in the checklist](https://github.com/MatsalakO/py-cat-and-dog-years/blob/develop/checklist.md#clean-code).

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested.

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!
